packages/@lwc/engine-server/src/__tests__/fixtures/dynamic-components/slots/scoped-slots/modules/x/test/test.js

```diff
@@ -2,4 +2,9 @@ import { LightningElement } from 'lwc';
 
 export default class Test extends LightningElement {
     static renderMode = 'light';
+
+    item = {
+        id: 'a',
+        name: 'b',
+    };
```

Comment:
Actually had to change the fixture because it seems `engine-server` silently fails:

```html
<x-dynamic-component>
  <template shadowrootmode="open">
    <x-test>
      a
      <!---->
      <!---->
      b
    </x-test>
  </template>
</x-dynamic-component>
```

while `ssr-compiler` will emit code that throws:

```
Cannot read properties of undefined (reading 'id')
```